app/controllers/friendships_controller.rb

```diff
@@ -15,10 +15,11 @@ def create
   end
 
   def update
-    friendship = Friendship.where(user_id: params[:id], friend_id: current_user.id)[0]
-    friendship.update(confirmed: true)
+    friendship1 = Friendship.where(user_id: params[:id], friend_id: current_user.id)[0]
+    friendship1.update(confirmed: true)
+    friendship2 = Friendship.create(user_id: current_user.id, friend_id: params[:id], confirmed: true)
```

Comment:
**REQUIRED**
* Everything connected with data processing should be encapsulated in models and used in controllers.
In this case, we can have a confirmed friendship method in the friendship model.
```ruby
def confirm_friendship
    self.update(confirmed: true)
    Friendship.create(user_id: self.friend_id, friend_id: self.user_id, confirmed: true)
  end
```
The update the update method to look like this.
```ruby
def update
    friendship1 = Friendship.where(user_id: params[:id], friend_id: current_user.id)[0]
    friendship2 = friendship1.confirm_friendship()

    if friendship1 && friendship2
      flash[:notice] = 'Friendship accepted!'
    else
      flash[:alert] = 'It was not possible to accept this friendship. Try again later.'
    end

    redirect_to users_path
  end
```
**Resources**
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright